packages/material-ui/src/OutlinedInput/NotchedOutline.js

```diff
@@ -125,8 +130,9 @@ const NotchedOutline = React.forwardRef(function NotchedOutline(props, ref) {
         }}
       >
         {/* Use the nominal use case of the legend, avoid rendering artefacts. */}
+        {/* notranslate needed while Google Translate will not fix zero-width space issue */}
         {/* eslint-disable-next-line react/no-danger */}
-        <span dangerouslySetInnerHTML={{ __html: '&#8203;' }} />
+        <span className="notranslate" dangerouslySetInnerHTML={{ __html: '&#8203;' }} />
```

Comment:
This branch is deprecated, we need to remove it before v5 stable

```suggestion
        <span dangerouslySetInnerHTML={{ __html: '&#8203;' }} />
```

Reply:
@oliviertassinari but can't we leave it for now? Because this part is the most important one

Reply:
This branch should never be reached anymore. If it's, it surfaces another root issue.